frontend/src/lib/blog.zod.ts

```diff
@@ -0,0 +1,26 @@
+import { z } from 'zod';
+
+export const FormDataSchema = z.object({
+  title: z.string().refine((value) => value.trim().split(/\s+/).length > 3, {
+    message: 'Oops! Title needs more spice. Give it at least 3 words.',
+  }),
+  authorName: z.string().refine((value) => value.trim().length >= 3 && value.trim().length <= 15, {
+    message: 'Hey, Your name should be more than 5 letters.',
+  }),
```

Comment:
We can't use "your name **should** be" emphasis on should, becuase he can be Raj, 
We can be like "C'mon your name cannot be less than 3 characters" ( if its less than 3 letters);
if greater than 15 say "Hey isn't it too big of a name, can you limit it to 15 characters" or improvise on this